### ISSUE TITLE
Adding a License to the Project

### ISSUE THREAD

**Issue by ghost** [facetoe/terminote2#16]
I know it's kind of annoying, but in order to avoid legal ambiguity and make the project look a little more 'complete' I think it would nice if you could chose an open source license for the project to use. The legal situation is kinda of odd if you don't.

> ...generally speaking, the absence of a license means that default copyright laws apply. This means that you retain all rights to your source code and that nobody else may reproduce, distribute, or create derivative works from your work. This might not be what you intend.

http://choosealicense.com/

I don't care which one you choose, and I'll help add the appropriate headers and such to source files. I just think it's something important to have.

**Comment by facetoe** [comment]
Good point. I'll have a look at the site you linked when I get home and choose a license to put terminote under.
